frontend/openchat-agent/src/services/user/user.client.ts

```diff
@@ -233,8 +232,9 @@ export class UserClient extends CandidService {
 
     createCommunity(
         community: CommunitySummary,
-        rules: AccessRules,
-        defaultChannels: string[]
+        rules: Rules,
+        defaultChannels: string[],
+        defaultChannelRules: Rules,
```

Comment:
Why do we need to pass these in given that there is no way in the UI to set them?